src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.TestLibrary/System.Text.Json.TestLibrary.Roslyn3.11.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="System.Text.Json.TestLibrary.targets" />
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\gen\System.Text.Json.SourceGeneration.Roslyn3.11.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="false" />
```

Comment:
I could have chosen just 1 copy of the generator to use, or even just checked in the generated code, but I created a parallel structure in case you find it interesting to add more tests.  Let me know if you think this is overkill.